src/wp-includes/class-wp-query.php

```diff
@@ -1516,7 +1539,11 @@ protected function parse_search_order( &$q ) {
 
 			// Sentence match in 'post_title'.
 			if ( $like ) {
-				$search_orderby .= $wpdb->prepare( "WHEN {$wpdb->posts}.post_title LIKE %s THEN 1 ", $like );
+				if ( wp_fulltext_search_enabled() ) {
+					$search_orderby .= $wpdb->prepare( "WHEN MATCH({$wpdb->posts}.post_title) AGAINST (%s IN BOOLEAN MODE) THEN 1 ", $q['s'] );
+				} else {
+					$search_orderby .= $wpdb->prepare( "WHEN {$wpdb->posts}.post_title LIKE %s THEN 1 ", $like );
```

Comment:
This mimics like behavior 1:1 which may be okay, but replacing all of that with relevancy that comes from `MATCH()` could potentially be better here.